maint.php

```diff
@@ -756,6 +757,10 @@ function schedules() {
 
 function thold_hosts($header_label) {
 	global $assoc_actions, $item_rows;
+	define('HOST_FILTER_LOC_ANY', '__any'); // internal value unlikely in data
+	define('HOST_FILTER_LOC_NONE', '__none');
+	define('HOST_FILTER_ANY', '-1');
+	define('HOST_FILTER_NONE', '0'); // Must be zero to match host table values for None
```

Comment:
Defines should be made global if they may be used by others.  For example, if these could be used in a URL from elsewhere, better to define them at a higher level rather than in a function.

Reply:
The constants have been moved.